.circleci/config/parameters.yml

```
@@ -0,0 +1,4 @@
+parameters:
+    run-database-tests:
```

```suggestion
    upgrade-directory-changed:
```
In my opinion, as for events with listener, it's the job/step decision to decide what is launched according to what is changed.
So I would just use the parameter as for an event: "this directory changed, deal with it".

---

Damn, I merged too fast and forgot to handle this feedback.

---

If you want to change the name of the parameter, leave me an :+1: emoji and I will open a new PR.